.env.example

```diff
@@ -59,10 +58,13 @@ TEST_ON_HARDHAT_NODE=
 #COSOUL_OPTIMISM_MNEMONIC=add a mnemonic for use deploying to optimism mainnet
 #OPTIMISM_RPC_URL=get_a_rpc_from_alchemy
 OPTIMISM_GOERLI_RPC_URL=get_a_rpc_from_alchemy
-REACT_APP_OPTIMISM_GOERLI_RPC_URL=get_a_rpc_from_alchemy_for_magic
+REACT_APP_ETH_MAINNET_API_KEY=get_ethereum_mainnet_api_key_from_alchemy
+REACT_APP_ETH_GOERLI_API_KEY=get_ethereum_goerli_api_key_from_alchemy
+REACT_APP_OPTIMISM_MAINNET_API_KEY=get_optimism_mainnet_api_key_from_alchemy
+REACT_APP_OPTIMISM_GOERLI_API_KEY=get_optimism_goerli_api_key_from_alchemy
 
 # used to fork mainnet when testing
-HARDHAT_ARCHIVE_RPC_URL=https://mainnet.infura.io/v3/your_infura_project_id
+HARDHAT_ARCHIVE_RPC_URL=get_ethereum_mainnet_rpc_url_from_alchemy
```

Comment:
we can just remove this completely now and just use the value of REACT_APP_ETH_MAINNET_API_KEY interpolated into a full right?

Reply:
I made the necessary changes to use REACT_APP_ETH_MAINNET_API_KEY instead